### ISSUE TITLE
REQ: Accessibility shortcut

### ISSUE THREAD

**Issue by sanctimoniousape** [chayanforyou/QuickBall#42]
Too many apps today are being written strictly for those who use gesture navigation, thus failing to allow space for the three-button navigation bar & it's gotten pretty annoying to have part of an app's screen inaccessible due to this issue. .  Also on the navigation bar (for those who activate it) is the accessibility shortcut in the bottom right corner. Long pressing it allows selecting the accessibility feature you want to use - such as screen magnifier, screen reader, etc. - while tapping on it activates the selected feature.

I was considering using this app to capitulate to all the inconsiderate programmers who think everybody loves gesture navigation by replacing the bottom navigation panel with it. Alas, this app doesn't support the accessibility shortcut, so I'm writing to request that you consider adding it (supporting long-presses as well so the desired utility can be selected first)

I can't help but note that it's kinda ironic that this app requires accessibility permissions to function, but doesn't actually provide the accessibility shortcut as an option. Hopefully, this can be changed. 

Thanks for taking the time to read my little whine, and hopefully I've not put you off enough to just deny or ignore it.  Thank you also for the app - I do look forward to seeing where you take it.

**Comment by chayanforyou** [comment]
Thanks for the detailed feedback. I understand your concern regarding the accessibility shortcut support. I’ll keep it in mind for future improvements. Glad to hear you’re enjoying the app!

**Comment by sanctimoniousape** [comment]
You have a minor in PR, huh?
